src-files/def-config.yaml

```diff
@@ -25,15 +25,24 @@ verification:
 
   # File prefixes, references in which should not be analyzed.
   notScanned:
-    # Github-specific files
+    # GitHub-specific files
     - .github/pull_request_template.md
     - docs/pull_request_template.md
 
-  # Glob patterns describing the files which do not physically exist in the repository
-  # but should be treated as existing nevertheless.
+    # GitLab-specific files
+    - .gitlab/merge_request_templates/
+
+  # Glob patterns describing the files which do not physically exist in the
+  # repository but should be treated as existing nevertheless.
```

Comment:
How is it used? Is there an example?

Reply:
Isn't it clear from the examples below?

Reply:
I am talking exactly about those examples. Where do we use `../../../pulls` and such?

Reply:
AFAIU it's some GitHub magic/hacks that I am not familiar with (or don't realize that).

Reply:
Ah hm, I was sure that we use such links in metatemplates, but we actually don't.

AFAIU it's the only way to refer to `issues` or `merge requests` page without mentioning the repository name. And Github/Gitlab requires such weird number of '..', no one knows why.